content/messaging/agent-chat-api/index.mdx

```diff
@@ -46,6 +46,10 @@ All the LiveChat OAuth2.0 access tokens have a prefix: `dal-` or `fra-`. This pr
 
 Summing up, if the user token starts with `fra-`, you should add the `X-Region: fra` header. If the token starts with `dal-`, you don’t have to specify the header.
 
+## Rate limits
+
+To prevent sending a massive number of requests, there are rate limits for methods' calls. After exceeding them, the requester will get a `Too many requests` error in a response to a specific request. After some time, the request will be unblocked so it can be sent again.
```

Comment:
```suggestion
To prevent sending a massive number of requests, there are rate limits for method calls. After exceeding those limits, the requester will get a `Too many requests` error in the response of a specific request. After some time, the request will be unblocked so it can be sent again.
```

Reply:
Shouldn't we specify, at least approximately, after how long the request will be unblocked?

Reply:
Unfortunately, we cannot specify that yet, these times can be different for each rate limit.